watchers/target_pod_handler.go

```diff
@@ -451,7 +452,7 @@ func (d DisruptionTargetHandler) buildNodeEventsToSend(oldNode corev1.Node, newN
 		case corev1.NodeRunning:
 			eventsToSend[v1beta1.EventTargetNodeRecoveredState] = true
 		case corev1.NodePending, corev1.NodeTerminated:
-			if oldNode.Status.Phase == corev1.NodeRunning {
+			if oldNode.Status.Phase == corev1.NodeRunning && d.disruption.Spec.Level != types.DisruptionLevelNode {
```

Comment:
(i know this is in draft, feel free to ignore) we can have nodeFailure disruptions at the pod level, it would be better to check if Spec.NodeFailure is defined

Reply:
Good know, I was not aware of this 🙈